packages/fxa-shared/db/models/auth/prune-tokens.ts

```diff
@@ -10,34 +10,17 @@ import { BaseAuthModel, Proc } from './base-auth';
  * Manages pruning of stale tokens.
  */
 export class PruneTokens extends BaseAuthModel {
-  /** Current state that the token pruner is in. */
-  public get state() {
-    return this._state;
-  }
-
-  /**
-   * Gets the timeout id for any pending prune operation.
-   */
-  public get currentTimeoutId() {
-    return this._timeoutId;
-  }
-
-  private _state: 'open' | 'pending' | 'active';
-  private _timeoutId: any | undefined;
-
   /**
    * Creates a token pruner
    * @param pruneInterval - A set interval at which to attempt prune operations
    * @param metrics - A statsd instance
    * @param log - A logger
    */
   constructor(
-    public readonly pruneInterval: number,
```

Comment:
Similarly, we don't need a prune interval, since we aren't worried about de-bouncing now.